sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitGrid.java

```diff
@@ -70,13 +71,15 @@ public VisitGrid(VisitCriteria criteria) {
 
 		((Column<VisitIndexDto, Date>) getColumn(VisitIndexDto.VISIT_DATE_TIME))
```

Comment:
Not all instances of SymptomsDto.XXX have been reverted here, please check again - it should only be used for SymptomsDto.I18N_PREFIX